src/Appwrite/Utopia/Database/Validator/Queries/Identities.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Appwrite\Utopia\Database\Validator\Queries;
+
+class Identities extends Base
+{
+    public const ALLOWED_ATTRIBUTES = [
```

Comment:
Might be worth adding index for token expiration

Reply:
I can add an index if you need to query for identities based on `providerAccessTokenExpiry`, but I'm hesitant to expose it on the API in case we decide to restructure the data. What do you think?

Reply:
It make sense to me to have the expiry, in which case you think we might want to hide it?

Reply:
@eldadfux, I really want identities to support other things besides oauth so there might be cases where the identity doesn't have a providerAccessTokenExpiry. For example, an additional email or phone.

Reply:
We could examine the approach we used for attributes where we have a base model that is extended. Attributes is also the only place where an array of models can contain models of different types. So, I think we might have the foundation for it.

Reply:
@Meldiron, @eldadfux, I've added providerAccessTokenExpiry as a filterable attribute.